src/Proxy.hs

```diff
@@ -1768,14 +1831,15 @@ putGTUDropR addrText = do
         "errorMessage" .= i18n i EMConfigurationError,
         "error" .= fromEnum InternalError
         ]
+    -- VH/FIXME: Should we supply the block address here from the calling function, or is it fine to use last finalized for each request?
     tryDrop addr dropType gtuDropData@GTUDropData{..} = do
```

Comment:
Should we perhaps supply the block address from the calling function, or is it fine to use `LastFinal` for each request as is already done?